spec/fixtures/files/watched/user@domain.com/owntracks.rec

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Real owntracks file structure looks like this: https://github.com/Freika/dawarich/blob/master/spec/fixtures/files/owntracks/2024-03.rec